.github/workflows/lint.yml

```diff
@@ -37,4 +37,26 @@ jobs:
           cd Docker/test/selenium 
           npm install
           npm run lint
+  lint_diagrams:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: c-py/action-dotenv-to-setenv@v2
+        with:
+          env-file: variables.env
+      - name: cache
+        uses: actions/cache@v2
+        with:
+          path: |
+            **/node_modules
+          key: npm-lint-cache
+      - name: lint the selenium javascript tests
+        run: |
+          cd diagrams
+          npm install
+          npm run lint
+      - name: test building diagram
+        run: |
+          cd diagrams
+          node make_overview.js
```

Comment:
Could be cool to diff the existing one with what is generated here to get an indication when this should be updated. but might be le overkill.

Reply:
This sounds exciting and like it wouldn't be too much work?

Reply:
Ugh, not really gave it a shot yesterday and there are several annoying things.

- Id gets changed every time you rebuild so need to swap that out.
- Widths and heights seem to change every time
- It's just one line so diff isn't to happy, need to do some md5 comparison or get wdiff which isnt' installed by default.